pom.xml

```diff
@@ -1139,35 +1120,34 @@
 
               <execution>
                 <phase>initialize</phase>
-                <id>install node and npm</id>
+                <id>install node and yarn</id>
                 <goals>
-                  <goal>install-node-and-npm</goal>
+                  <goal>install-node-and-yarn</goal>
                 </goals>
                 <configuration>
                   <nodeVersion>v${node.version}</nodeVersion>
-                  <npmVersion>${npm.version}</npmVersion>
-                  <!-- Use the pre-download node and npm packages. See download-maven-plugin config above. -->
-                  <downloadRoot>${project.baseUri}target/frontend/</downloadRoot>
+                  <yarnVersion>v${yarn.version}</yarnVersion>
                 </configuration>
               </execution>
 
               <execution>
                 <phase>initialize</phase>
-                <id>npm install</id>
+                <id>yarn install</id>
```

Comment:
There is. The diff here is very poor unfortunately. Please have a look at 0a56287 as it adds the functionality for yarn without touching the existing npm functionality.